Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -248,18 +251,25 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
       .skipNil()
       .takeWhen(self.dataSourceLoadedProperty.signal)
 
-    // TODO: Call blocking GraphQL mutation
-    self.userBlocked = self.blockUserProperty.signal.map { true }
+    let blockUserEvent = self.blockUserProperty.signal
+      .map(BlockUserInput.init(blockUserId:))
+      .switchMap { input in
+        AppEnvironment.current.apiService
+          .blockUser(input: input)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
```

Comment:
Just for my own edition, what's up with `ksr_delay` everywhere when we make network requests?